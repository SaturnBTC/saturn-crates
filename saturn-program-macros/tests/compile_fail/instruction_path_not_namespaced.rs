use saturn_program_macros::saturn_program;

mod instruction {
    #[derive(borsh::BorshSerialize, borsh::BorshDeserialize)]
    pub enum Instr {
        Dummy,
    }
}

#[saturn_program(instruction = "Instr")]
mod handlers {}

fn main() {} 