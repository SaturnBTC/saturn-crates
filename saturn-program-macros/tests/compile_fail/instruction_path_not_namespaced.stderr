error: instruction path must be namespaced (e.g. `crate::ix::MyInstr`) â€“ add a module prefix
  --> tests/compile_fail/instruction_path_not_namespaced.rs:10:32
   |
10 | #[saturn_program(instruction = "Instr")]
   |                                ^^^^^^^
